src/main/java/net/kodehawa/mantarobot/commands/MoneyCmds.java

```diff
@@ -446,13 +449,18 @@ public void call(GuildMessageReceivedEvent event, String content, String[] args)
                                 .optArg("index", r.desc("money"));
 
                 if(args.length > 0 && (args[0].equalsIgnoreCase("lvl") || args[0].equalsIgnoreCase("level"))) {
-                    Cursor<Map> m = r.table("players")
-                            .orderBy()
-                            .optArg("index", r.desc("level"))
-                            .filter(player -> player.g("id").match(pattern))
-                            .map(player -> player.pluck("id", "level"))
-                            .limit(15)
-                            .run(MantaroData.conn(), OptArgs.of("read_mode", "outdated"));
+                    Cursor<Map> m;
+
+                    try(Connection conn = r.connection().hostname(c.dbHost).port(c.dbPort).db(c.dbDb).user(c.dbUser, c.dbPassword).connect()) {
```

Comment:
Make a utility method for getting connections just so you don't keep repeating yourself?

Reply:
Might also be a good idea to pass a callback for a lot of stuff like this just so you don't have to think about remembering the try with resources everywhere